ui/package.json

```diff
@@ -28,33 +28,33 @@
     "vue": "^3.2.25"
   },
   "devDependencies": {
-    "@featherds/app-bar": "0.10.14",
-    "@featherds/app-layout": "0.10.14",
-    "@featherds/autocomplete": "0.10.14",
-    "@featherds/button": "0.10.14",
-    "@featherds/checkbox": "0.10.14",
-    "@featherds/chips": "0.10.14",
-    "@featherds/date-input": "0.10.14",
-    "@featherds/dialog": "0.10.14",
-    "@featherds/drawer": "0.10.14",
-    "@featherds/dropdown": "0.10.14",
-    "@featherds/expansion": "0.10.14",
-    "@featherds/icon": "0.10.14",
+    "@featherds/app-bar": "0.10.15",
+    "@featherds/app-layout": "0.10.15",
+    "@featherds/autocomplete": "0.10.15",
+    "@featherds/button": "0.10.15",
+    "@featherds/checkbox": "0.10.15",
+    "@featherds/chips": "0.10.15",
+    "@featherds/date-input": "0.10.15",
+    "@featherds/dialog": "0.10.15",
+    "@featherds/drawer": "0.10.15",
+    "@featherds/dropdown": "0.10.15",
+    "@featherds/expansion": "0.10.15",
+    "@featherds/icon": "0.10.15",
     "@featherds/input": "0.10.15",
-    "@featherds/list": "0.10.14",
-    "@featherds/megamenu": "0.10.14",
-    "@featherds/navigation-rail": "0.10.14",
-    "@featherds/pagination": "0.10.14",
-    "@featherds/progress": "0.10.14",
-    "@featherds/protected-input": "0.10.14",
-    "@featherds/radio": "^0.10.14",
-    "@featherds/select": "0.10.14",
-    "@featherds/snackbar": "0.10.14",
-    "@featherds/styles": "0.10.14",
-    "@featherds/table": "0.10.14",
-    "@featherds/tabs": "0.10.14",
-    "@featherds/tooltip": "0.10.14",
-    "@featherds/utils": "0.10.14",
+    "@featherds/list": "0.10.15",
+    "@featherds/megamenu": "0.10.15",
+    "@featherds/navigation-rail": "0.10.15",
+    "@featherds/pagination": "0.10.15",
+    "@featherds/progress": "0.10.15",
+    "@featherds/protected-input": "0.10.15",
+    "@featherds/radio": "^0.10.15",
+    "@featherds/select": "0.10.15",
+    "@featherds/snackbar": "0.10.15",
+    "@featherds/styles": "0.10.15",
+    "@featherds/table": "0.10.15",
+    "@featherds/tabs": "0.10.15",
+    "@featherds/tooltip": "0.10.15",
+    "@featherds/utils": "0.10.15",
```

Comment:
I guess they found a couple bugs in .15, as they pushed to .17 already. But probably won't affect you here so can do it later. I usually run `yarn upgrade --latest` to do all packages at once